src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.Netcore.csproj

```diff
@@ -27,7 +27,7 @@
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="23.0.1" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="2.4.2" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.5.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.6.0-preview" />
```

Comment:
@MikhailTryakhov The test issue is caused by this change in your SDK: https://github.com/Azure/azure-sdk-for-net/commit/b5365ba9e8d041180c8526d36063eb0b1dd67982#diff-613c66b988b903b73257bbf87081efbdR301 

Given that this is a breaking change, you likely want to fix this in your SDK rather than change the call for the new parameter order in powershell.

Reply:
yeah, I see... it'll take some time to update all of the repos... so can we use another way and release PS  later?